src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.ConfigUpdaters/AppSettingsConfigUpdater.cs

```diff
@@ -51,7 +51,7 @@ public async Task<bool> ApplyAsync(IUpgradeContext context, ImmutableArray<Confi
             var project = context.CurrentProject.Required();
 
             // Determine where appsettings.json should live
-            var appSettingsPath = Path.Combine(project.Directory ?? string.Empty, AppSettingsJsonFileName);
+            var appSettingsPath = Path.Combine(project.FileInfo.DirectoryName, AppSettingsJsonFileName);
```

Comment:
DirectoryName can be null, I think, so we still need the ?? here and in the other similar places it was removed.

Reply:
There was no nullable error, so I assumed it was ok... we're targeting .net standard 2.0 which doesn't have those marked